### PR TITLE
Allow additional properties for ResourceConfig

### PR DESCRIPTION
**Pull request by marcust**

### CODE REVIEW

File: vertx-mod-jersey/src/main/java/com/englishtown/vertx/jersey/impl/DefaultJerseyConfigurator.java [hunk 1]
```diff
@@ -263,6 +264,13 @@ protected ResourceConfig getResourceConfig() {
             }
         }
 
+        final JsonObject resourceConfigAdditions = config.getObject(CONFIG_RESOURCE_CONFIG);
+        if ( resourceConfigAdditions != null ) {
```
Comment (adrianluisgonzalez) [inline vertx-mod-jersey/src/main/java/com/englishtown/vertx/jersey/impl/DefaultJerseyConfigurator.java]: Do you mind updating this to be consistent with the code style in the rest of the project?

**Comment by adrianluisgonzalez** [comment]
Thanks for the contribution.  Small comment about being consistent with the existing code style.  Looks good apart from that.

**Comment by marcust** [comment]
Better this way?

**Comment by adrianluisgonzalez** [comment]
Perfect, thanks.

**Pull request merged**